atlasdb-ete-tests/Dockerfile

```diff
@@ -1,6 +1,23 @@
-FROM frolvlad/alpine-oraclejdk8:slim
-
-RUN apk update && apk add libstdc++ bash wget
+FROM centos:centos7
+
+MAINTAINER hsaraogi@palantir.com <Himangi Saraogi>
+
+# Default to UTF-8 file.encoding
+ENV LC_ALL en_US.UTF-8
+
+ENV JAVA_HOME /usr/java/jdk1.8.0_102/jre
+ENV JAVA_VERSION 8u102
+ENV RPM_VERSION 8u102-b14
+
+# - install java8 jdk
+RUN yum install -y wget && \
+    wget \
+      --no-cookies \
+      --no-check-certificate \
+      --header "Cookie: gpw_e24=http%3A%2F%2Fwww.oracle.com%2F; oraclelicense=accept-securebackup-cookie" \
+      "http://download.oracle.com/otn-pub/java/jdk/${RPM_VERSION}/jdk-${JAVA_VERSION}-linux-x64.rpm" && \
+    yum localinstall -y "jdk-${JAVA_VERSION}-linux-x64.rpm" && \
+    rm -f "jdk-${JAVA_VERSION}-linux-x64.rpm"
 
 ENV DOCKERIZE_VERSION v0.2.0
 
```

Comment:
@tpetracca @jeremyk-91 This doesnt seem to add any overall build time overhead, maybe because this is run on a container with fewer tests. But I dont see an argument in investing on further optimizing this test.

I tried `jre8` and installing `ps` on alpine, both of them will make the tests not pass.